src/Display/Modal/Modal.tsx

```diff
@@ -61,6 +68,25 @@ const Modal = (props: Props) => {
       document.removeEventListener('keydown', escapeKeyEventListener, false);
   }, [isVisible, onClose]);
 
+  React.useLayoutEffect(() => {
+    if (footerElementCount > 1) {
+      const checkIsFooterChildrenInMultiLines = () => {
```

Comment:
@rosslo ,  should this function be outside the hook. It's just weird to me that inside the hook there's a function in the hook, then it's called and then an event listener is set.

Reply:
If we put this function be outside the hook, we will need to wrap useCallback for it, I think we can just declare it inside useLayoutEffect to save a bit performance.